Libraries/Azure.psm1

```diff
@@ -505,25 +510,42 @@ Function Create-ResourceGroup([string]$RGName, $location, $CurrentTestData) {
     return $retValue
 }
 
-Function Create-ResourceGroupDeployment([string]$RGName, $location, $TemplateFile, $UseExistingRG, $ResourceCleanup) {
-    $FailCounter = 0
-    $retValue = "False"
+Function Create-ResourceGroupDeployment([string]$RGName, $TemplateFile, $UseExistingRG, $ResourceCleanup) {
+    $retValue = $false
     $ResourceGroupDeploymentName = "eosg" + (Get-Date).Ticks
-    While (($retValue -eq $false) -and ($FailCounter -lt 1)) {
```

Comment:
don t remove failcounter, it can be used later with another value, like 3 retries.

Reply:
if we need it later we should add it back then, there's no sense in having code that does nothing more than adding more lines.